frontend/src/pages/modelRegistry/screens/utils.ts

```diff
@@ -172,15 +191,25 @@ export const filterRegisteredModels = (
         const matchesModel =
           rm.name.toLowerCase().includes(searchLower) ||
           (rm.description && rm.description.toLowerCase().includes(searchLower)) ||
-          getLabels(rm.customProperties).some((label) => label.toLowerCase().includes(searchLower));
+          (rm.customProperties &&
+            getLabels(rm.customProperties).some((label) =>
+              label.toLowerCase().includes(searchLower),
+            )) ||
+          (rm.modelSourceName && rm.modelSourceName.toLowerCase().includes(searchLower)) ||
+          (rm.modelSourceId && rm.modelSourceId.toLowerCase().includes(searchLower)) ||
+          (rm.modelSourceGroup && rm.modelSourceGroup.toLowerCase().includes(searchLower));
```

Comment:
Same here, we can't really filter models by these without looking up artifacts for every version which isn't feasible.